pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -62,6 +63,29 @@ def test_neg_latent_time(self, mock_choose):
             test_sweep = pe.sweep.HostProgressionSweep()
             test_sweep._set_latent_time(self.person1, current_time)
```

Comment:
I think its nicer to only have one line in the `with self.assertRaises` clause to be precise on where the error should be called (and prevent false positives) - can L63 be moved outside this clause?